content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -25,7 +25,21 @@ it succeeds, making sure those probes don't interfere with the application start
 This can be used to adopt liveness checks on slow starting containers, avoiding them
 getting killed by the kubelet before they are up and running.
 
+{{< caution >}}
+Liveness probes can be a powerful way to recover from application failures, but
+they should be used with caution. Liveness probes must be configured carefully
+to ensure that they truly indicate unrecoverable application failure, for example a deadlock.
+A common pattern for liveness probes is to use the same low-cost HTTP endpoint
+as for readiness probes, but with a higher `failureThreshold`. This ensures that the pod
+is observed as not-ready for some period of time before it is hard killed.
```

Comment:
This should not be part of the caution?

Reply:
Do I put this into note a section?

Reply:
My feeling is that this can be in plain text. It is a hint, not something users must follow or must be alerted for.

Reply:
Alright then I will modify the changes